src/libmongoc/doc/mongoc_client_encryption_encrypt_opts_set_text_opts.rst

```diff
@@ -0,0 +1,31 @@
+:man_page: mongoc_client_encryption_encrypt_opts_set_text_opts
+
+mongoc_client_encryption_encrypt_opts_set_text_opts()
+=====================================================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+   void
+   mongoc_client_encryption_encrypt_opts_set_text_opts(
+      mongoc_client_encryption_encrypt_opts_t *opts,
+      const mongoc_encrypt_text_opts_t *text_opts);
+
+.. versionadded:: 2.2.0
+
+Sets the text search encryption options for explicit encryption.
```

Comment:
Add documentation to the text API pages to indicate experimental:

> The "TextPreview" algorithm is in preview and should be used for experimental workloads only. These features are unstable and their security is not guaranteed until released as Generally Available (GA). The GA version of these features may not be backwards compatible with the preview version.

An RST replacement can be used to avoid repeated text ([example](https://github.com/mdb-ad/mongo-c-driver/blob/b1ba169212cb9314376bc2652df11d327ad07e43/src/libmongoc/doc/conf.py#L190-L191)).